crates/oxc_semantic/src/builder.rs

```diff
@@ -537,6 +539,10 @@ impl<'a> SemanticBuilder<'a> {
         }
     }
 
+    fn add_node_id(&mut self) {
```

Comment:
let's change this to `add_current_node_id_to_current_scope`. `add_node_id` is too vague.